include/Ark/REPL/Repl.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef ark_repl
+#define ark_repl
+
+#include <iostream>
+#include <sstream>
+
+#include <Ark/Constants.hpp>
+#include <Ark/Compiler/Compiler.hpp>
+#include <Ark/VM/VM.hpp>
+#include <Ark/VM/State.hpp>
+
+namespace Ark {
```

Comment:
I'm a bit of a maniac, but it would be better to stick with the current coding style (which you used here and there in Repl.cpp) by putting the `{` on a new line :)

Reply:
No, not a problem at all. This is your project after all. :)